### ISSUE TITLE
Support for quotes and replies

### ISSUE THREAD

**Issue by DarremMolko** [TBXark/ChatGPT-Telegram-Workers#239]
Hi. I have been using this bot for a couple of months and I really like it. One thing I think would be nice is if the bot could interact with quotes and replies. For example if I quote or reply to someone else's message with "@mybot translate this" it should translate the quoted/replied message. Is this possible? Thank you in advance.

**Comment by TBXark** [comment]
Do you mean to use it in a group chat?

**Comment by DarremMolko** [comment]
Yes, that's why I said while quoting or replying to someone else's message.

**Comment by TBXark** [comment]
Sounds like a good idea,I'll try tomorrow.

**Comment by TBXark** [comment]
The dev branch has already added this feature, you need to add an environment variable to activate it. `EXTRA_MESSAGE_CONTEXT = "true"`

**Comment by TBXark** [comment]
<img width="604" alt="image" src="https://github.com/TBXark/ChatGPT-Telegram-Workers/assets/9513891/81e31505-d584-4a63-8a7c-aaf9bbf5f44f">

**Comment by DarremMolko** [comment]
Thank you! It seems to be working as intended.
